server/controllers/chartsController/helpers.test.js

```diff
@@ -171,6 +172,7 @@ describe('chartsController - helpers', () => {
         MA: { count: 10, targetTotal: undefined },
         NY: { count: 35, targetTotal: 35 },
         Foo: { count: 5, targetTotal: 20 },
+        empty: undefined,
```

Comment:
where is this empty study coming from? It isn't in the list of allowed studies.

Reply:
That's what a new site is added, it has data but that data does not match any of the values we are looking for

Reply:
I think we can remove this. If a new site is added the study totals will look like `empty: { count: 0, targetTotal: undefined }`.